src/analysis/complex/conformal.lean

```diff
@@ -0,0 +1,136 @@
+/-
+Copyright (c) 2021 Yourong Zang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yourong Zang
+-/
+import analysis.complex.isometry
+import analysis.calculus.conformal
+
+/-!
+# Complex Conformal Maps
+
+We prove the sufficient and necessary conditions for a complex function to be conformal.
```

Comment:
This module docstring needs to be updated now!  "real-linear map between complex vector spaces" perhaps, rather than "complex function".